recipes/gpmdp.mdwn

```diff
@@ -0,0 +1,21 @@
+# Google Play Music Desktop Player watcher
+
+This widget can be used to display the current track information and state of [Google Play Music Desktop Player](https://www.googleplaymusicdesktopplayer.com/).
+
+1. Download `gpmdp.lua` [here](https://awesomewm.org/recipes/gpmdp.lua)
+2. Add your customization to `gpmdp.lua` starting at line 87
+    > for a in depth example, see [here](https://github.com/gregflynn/dotsanity/blob/master/awesome/widgets/gpmdp.lua)
```

Comment:
Remove the line, then. Example usage will be fine for beginners.

Once you do this last commit, before pushing, squash the previous two.